apps/test/unit/templates/CodeWorkspaceTest.js

```diff
@@ -5,7 +5,7 @@ import {UnconnectedCodeWorkspace as CodeWorkspace} from '../../../src/templates/
 import {singleton as studioAppSingleton} from '@cdo/apps/StudioApp';
 import sinon from 'sinon';
 import ShowCodeToggle from '@cdo/apps/templates/ShowCodeToggle';
-
+import {workspaceAlertTypes} from '@cdo/apps/code-studio/projectRedux';
 describe('CodeWorkspace', () => {
```

Comment:
nit: can you keep the line break between import and start of tests?

Reply:
Sure!